app/2.0/nav.yaml

```diff
@@ -193,8 +190,13 @@
   - title: Service worker
     path: /2.0/toolbox/service-worker
     indent: True
+  - endheader: true
+  - header: Deploy
```

Comment:
Still need to add a link for PRPL here. I merged there build PR, so you could move that over now. Or do it in a separate PR if you prefer.